models/User/AbstractUser.php

```diff
@@ -261,12 +262,19 @@ public function delete()
 
         // now delete the current user
         $this->getDao()->delete();
-        
+
         $cacheKey = 'user_' . $this->getId();
         if (RuntimeCache::isRegistered($cacheKey)) {
             RuntimeCache::set($cacheKey, null);
         }
 
+        if ($parentUserId && $parentUserId > 1) {
```

Comment:
to avoid clearing the system user, seeing 
https://github.com/pimcore/pimcore/pull/15609/files?w=1#diff-33922f1600adee6c57d7b56216b6fc1188e2ffef899a5ee79eaed8414edb352fR242-R243